src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/EvidenceUploadApplicantHandlerTest.java

```diff
@@ -604,10 +605,10 @@ void shouldBreakWhenThereIsAnyCaseBundlesWithNullCreatedDate() {
         CallbackParams params = callbackParamsOf(caseData, ABOUT_TO_SUBMIT);
 
         // When: handler is called
-        // Then: an exception is thrown
-        assertThrows(NullPointerException.class, () -> {
-            handler.handle(params);
-        });
+        var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
+        CaseData updatedData = mapper.convertValue(response.getData(), CaseData.class);
+        // Then: applicant docs uploaded after bundle should return size 0
+        assertThat(updatedData.getApplicantDocsUploadedAfterBundle()).isNull();
```

Comment:
If you had to modify a test, it means that the behaviour has changed.

I seem to recall we added this test specifically.

Please don't alter the logic ad this point in time. The ticket has already been approved. If you alter the logic, test will have to be re-run and PO approval sought after again.

Restore this test and the related logic as before (except for any potential short-circuiting, which shouldn't affect the test anyway).